Resources/config/events_deferring.yml

```diff
@@ -3,6 +3,11 @@ parameters:
   event_dispatcher.class: Smartbox\Integration\FrameworkBundle\Tools\EventsDeferring\EventDispatcher
 
 services:
+  event_dispatcher:
+    class: "%event_dispatcher.class%"
```

Comment:
What we can do here is is replace ```class: "%event_dispatcher.class%"``` for ```class: Smartbox\Integration\FrameworkBundle\Tools\EventsDeferring\EventDispatcher``` because the mininum version of symfony we are supporting is 2.8 and defining the class as ```parameter``` was the way to override the service until symfony 2.6 (see https://symfony.com/doc/2.6/cookbook/bundles/override.html#services-configuration)